packages/playwright/tests/axe-playwright.spec.ts

```diff
@@ -80,7 +84,7 @@ describe('@axe-core/playwright', () => {
     });
 
     it('returns correct results metadata', async () => {
-      await page.goto(`${addr}/external/index.html`);
+      const res = await page.goto(`${addr}/external/index.html`);
```

Comment:
`res` isn't used in the test. surprised that didn't throw a lint error for unused variable.